napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -23,11 +46,107 @@ def __init__(self, viewer: napari.Viewer):
         # 4 annotations
         # path = str(Directories.get_assets_dir() / "sample.json")
         # 8 annotations
-        path = str(Directories.get_assets_dir() / "sample2.json")
-        data = json.load(open(path))
+        path: str = str(Directories.get_assets_dir() / "sample2.json")
+        self.annot_data: Dict[str, Dict[str, str]] = json.load(open(path))
         # open in add mode
         # self.view = AnnotatorView(napari.Viewer(), data)
         # open in view mode
-        self.view = AnnotatorView(viewer, data, mode=AnnotatorViewMode.VIEW)
-        # open in annotate mode
-        # self.view = AnnotatorView(viewer, data, mode=AnnotatorViewMode.ANNOTATE)
+        self.view: AnnotatorView = AnnotatorView(
+            viewer, self, mode=AnnotatorViewMode.VIEW
+        )
+        self.view.render_annotations(self.annot_data)
+        self.view.show()
+        self.curr_img: Dict[str, str] = None
+        self.csv_name: str = None
+
+        self.annotation_dict: Dict[str, (List[str], List[str])] = {}
+
+    def set_csv_name(self, name: str):
+        """Set csv file name for writing."""
+        self.csv_name = name
+
+    def stop_annotating(self):
+        """Reset values from annotating and change mode to VIEW."""
+        self.view.set_curr_index(None)
```

Comment:
For all these function calls with `None`, do these values get initialized later? Passing none as a parameter works, but seems pretty weird to me style wise.  It'd be simpler to create a default value for that parameter (you can pass in no params, and it will default to None) https://www.geeksforgeeks.org/default-arguments-in-python/

So for `self.view.set_curr_index()` you'd do something like this:
```
def set_curr_index(self, num: Optional[int] = None]):
```
You have to change the type to make it nullable